assets/airports/kchs.json

```diff
@@ -0,0 +1,10037 @@
+        "LOFTI": ["N32d41m08.995", "W80d20m38.303"],
+        "LOIRN": ["N33d00m41.199", "W79d43m21.785"],
+        "LUTHR": ["N33d08m04.828", "W80d12m58.461"],
+        "MAMAE": ["N33d04m23.702", "W80d10m11.485"],
+        "MLTRE": ["N33d20m08.860", "W80d15m18.304"],
+        "MRISS": ["N33d01m08.941", "W80d04m19.226"],
+        "MRPIT": ["N34d26m05.165", "W79d01m44.857"],
+        "MURAY": ["N32d52m23.106", "W80d11m10.900"],
+        "MYERS": ["N33d01m41.314", "W80d04m02.681"],
+        "NALVE": ["N34d31m53.947", "W80d37m35.793"],
+        "NINJA": ["N34d31m53.947", "W80d37m35.793"],
+        "ONBEE": ["N34d44m24.301", "W80d37m35.303"],
+        "OSPRI": ["N33d08m10.967", "W80d23m18.004"],
+        "PALLI": ["N33d13m14.063", "W79d56m31.049"],
+        "PAMJE": ["N33d03m02.428", "W80d18m23.579"],
+        "PIPPY": ["N32d54m09.069", "W80d14m05.463"],
+        "PLMTO": ["N33d12m00.233", "W79d22m29.128"],
+        "POETS": ["N33d08m04.664", "W80d12m59.634"],
+        "RAPZZ": ["N34d15m03.368", "W78d29m16.740"],
+        "RECHY": ["N32d51m56.824", "W80d14m28.348"],
+        "RIKKI": ["N33d12m34.992", "W79d40m24.252"],
+        "ROHLO": ["N32d54m01.133", "W79d54m49.136"],
+        "RSRVE": ["N33d12m34.546", "W79d40m23.133"],
+        "SAV":   ["N32d08m46.616", "W81d11m56.676"],
+        "SEAMS": ["N33d08m10.684", "W80d23m17.406"],
+        "SHRMP": ["N32d42m43.217", "W79d53m46.039"],
+        "SNOBB": ["N32d53m45.127", "W79d53m37.381"],
+        "SOGTY": ["N32d51m00.023", "W80d00m01.115"],
+        "STNNS": ["N32d49m29.521", "W79d58m08.461"],
+        "STUNO": ["N32d58m31.516", "W80d53m36.112"],
+        "SWPFX": ["N33d22m59.302", "W79d53m48.569"],
+        "TAWKE": ["N31d15m56.877", "W81d12m51.189"],
+        "TRTLS": ["N33d13m58.362", "W80d40m24.218"],
+        "TYLOR": ["N32d52m14.612", "W80d11m27.582"],
+        "UNJAM": ["N34d12m56.552", "W80d48m10.587"],
+        "UZADY": ["N32d46m18.656", "W80d02m47.093"],
+        "VAN":   ["N33d28m29.355", "W80d26m54.449", "Vance"],
+        "WANDO": ["N33d02m03.469", "W79d50m26.905"],
+        "WBODY": ["N34d08m54.495", "W81d37m54.319"],
+        "WILIT": ["N33d29m30.797", "W79d27m38.471"],
+        "WOGIB": ["N33d37m59.033", "W82d18m43.983"],
+        "ZUNSU": ["N32d49m33.581", "W80d04m36.975"]
+    },
+    "runways": [
+        {
+            "name": ["3", "21"],
+            "end": [
+                ["N32d53m05.173", "W80d02m43.914", "45ft"],
+                ["N32d54m08.270", "W80d02m10.208", "45ft"]
+            ],
+            "ils": [false, false]
+        },
+        {
+            "name": ["15", "33"],
+            "end": [
+                ["N32d54m46.761", "W80d02m53.168", "45ft"],
+                ["N32d53m31.386", "W80d01m56.156", "45ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "LGRHD3": {
+            "icao": "LGRHD3",
+            "name": "Logger head Three",
+            "altitude": 4000,
+            "rwy": {
```

Comment:
Since LGRHD is a common point for all runways, it should be moved into the body section. Similarly, GIPPL is defined in the exitPoint so it shouldn't be in the runway section also.